newIDE/app/src/Utils/GDevelopServices/Example.js

```diff
@@ -14,6 +14,8 @@ export type ExampleShortHeader = {|
   authorIds?: Array<UserPublicProfileSearch>,
   previewImageUrls: Array<string>,
   gdevelopVersion: string,
+  // Used to highlight text when searching for keywords in examples.
+  matches?: Array<{| key: string, indices: number[][] |}>,
```

Comment:
I'm a bit concerned by this, I wonder if we can rather store the search results as `Array<{|item: SearchItem, matches: ...|}>` in the extension store context and use it like this rather than polluting Example.